mistletoe/html_attributes_renderer.py

```diff
@@ -0,0 +1,256 @@
+"""
+HTML renderer for mistletoe.
+"""
+
+import html
+import re
+from itertools import chain
+from urllib.parse import quote
+from mistletoe import block_token
+from mistletoe import span_token
+from mistletoe.block_token import HTMLBlock
+from mistletoe.span_token import HTMLSpan
+from mistletoe.base_renderer import BaseRenderer
+
+
+class HTMLAttributesRenderer(BaseRenderer):
```

Comment:
Consider inheriting from the HTMLRenderer rather than the BaseRenderer. Maybe you can remove some code duplication that way?

Reply:
See (https://github.com/miyuchina/mistletoe/pull/170/commits/e68273d0395889f17b8bfc46a49a235555320bc6)